src/test/java/org/jenkinsci/plugins/docker/workflow/WithContainerStepTest.java

```diff
@@ -256,7 +256,6 @@
         });
     }
 
-    @Ignore("TODO no fix yet")
```

Comment:
Please add an `Assume` in the test so that we can skip this test when on earlier versions of Docker, or else this’ll fail in CI, etc.